### PR TITLE
Segments:  opacity slider and visibile toggle

### PR DESCRIPTION
**Pull request by PaulHax**
image

### CODE REVIEW

File: src/store/segmentGroups.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ import {
 import { FileEntry } from '../io/types';
 import { ensureSameSpace } from '../io/resample/resample';
 import { useDICOMStore } from './datasets-dicom';
+import { vi } from 'vitest';
```
Comment (floryst) [inline src/store/segmentGroups.ts]: Unused import

**Review by floryst — APPROVED** [review]
LGTM with one unused import comment.

I re-ran the test in case it's transient, but if it fails again we'll take a look.

File: src/store/segmentGroups.ts (hunk 1 above)
Comment (PaulHax) [reply src/store/segmentGroups.ts]: got it!

**Comment by PaulHax** [comment]
Yes, the e2e layer test got flakey on ubuntu couple weeks ago.

**Pull request merged**